graylog2-web-interface/yarn.lock

```diff
@@ -7754,7 +7754,7 @@ eslint-config-airbnb@19.0.4:
     eslint-config-prettier "10.1.1"
     eslint-import-resolver-webpack "0.13.10"
     eslint-plugin-compat "6.0.2"
-    eslint-plugin-graylog "file:packages/eslint-plugin-graylog"
+    eslint-plugin-graylog "file:../../../../Library/Caches/Yarn/v6/npm-eslint-config-graylog-1.3.0-dd227ac5-c0f2-4514-a089-8eb670a79e7c-1741014568138/node_modules/eslint-plugin-graylog"
```

Comment:
We need to revert the change on the yarn.lock, we only update it when we do change in package, like update or add a new one.

Reply:
Oops, just fixed pushed this fix!